src/TodoListItem.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+function TodoListItem(props) {
+  // Destructuring props to get individual properties
+  const { title } = props.todo;
+
+  // Your component logic goes here
+
+  return (
+    <li>{title}</li>
+  );
+}
```

Comment:
So all of this works totally fine, but as an alternative I'd like to suggest the following

```suggestion
function TodoListItem(props) {
  return (
    <li>{props.todo.title}</li>
  );
}
```

It's just a bit more succinct but is functionally identical.

Reply:
@GeoSauer but doest this line make it cleaner?

const { title } = props.todo;

Reply:
Eh I'd argue it's a matter of opinion/style. This suggestion is more just based on what the course has covered so far and what the next assignment covers, which is destructuring.

Reply:
I had made all the corrections you asked and resubmitted the lesson without merging it